dipy/workflows/reconst.py

```diff
@@ -738,6 +745,12 @@ def run(self, input_files, bvalues_files, bvectors_files, mask_files,
         mask_files : string
             Path to the input masks. This path may contain wildcards to use
             multiple masks at once. (default: No mask used)
+        fit_method : string, optional
+            can be one of the following:
+            'WLS' for weighted least squares
+            'LS' or 'OLS' for ordinary least squares
+            'NLLS' for non-linear least-squares
+            'RT' or 'restore' or 'RESTORE' for RESTORE robust tensor fitting
```

Comment:
I don't believe that DKI has a RESTORE option.
```suggestion
```

Reply:
Good catch and Thanks! I just addressed your comment.